website/versioned_docs/version-4.x/getting-started.md

```diff
@@ -92,6 +92,13 @@ Next, we need to link these libraries. The steps depends on your React Native ve
   # npm install --save-dev jetifier
   ```
 
+  Next, add these two lines in `gradle.properties` under `android` folder
```

Comment:
cc @mikehardy can you verify if these instructions are correct?

Reply:
@satya164 yes they look correct to me as a start, but I would mention that you also want to add a call to actually jetify in the postinstall

```suggestion
  Then ensure jetifier runs after all package installs by adding a call to `npx jetify` in the `postinstall` lifecycle script of your `package.json` file, and make sure you have run `npx jetify` at least once prior to attempting to build for android.


  Next, add these two lines in `gradle.properties` under `android` folder
```

Or it might be better to just link to the jetifier docs? That would be https://github.com/mikehardy/jetifier#to-jetify--convert-node_modules-dependencies-to-androidx

Reply:
@mikehardy we do mention it below:

![Screenshot 2019-09-30 at 5 18 55 PM](https://user-images.githubusercontent.com/1174278/65892332-6039fc80-e3a6-11e9-96ad-762bec806b8e.png)

I'd emphasise that this part of the setup instructions is for **React Native 0.59 and lower** which didn't include `androidx`. That's why I'm not sure about this part.

Reply:
It is actually very difficult to be prescriptive for jetifier. It is possible to use jetifier in the forward or reverse direction on RN59 and it depends on what the project needs. Necessary complexity to be flexible unfortunately - the full 2x2 matrix of "library is AndroidX or support libraries / project is AndroidX or support libraries" is handled. So I would counsel to just expose the information "this library has converted to AndroidX, if you need special handling in your project for either AndroidX or Android support libraries, please consult the relevant jetifier documentation" or similar

A bit vague, sorry

Reply:
@satya164 But I think the instructions for setting up `jetifier` mentioned in the Doc for **React Native 0.59 and lower** isn't working without adding these config values inside gradle.properties these days.
![image](https://user-images.githubusercontent.com/8683960/65898956-17953a00-e3cc-11e9-9a09-b65638e5e2aa.png)

Reply:
but, that's only for people that want to *go forward* to AndroidX. It is vital for that. If people still want to go reverse, those are the exact wrong things to put there as they are part of an AndroidX transition (they tell the gradle plugin to do the jetify transformation to AndroidX on all static artifacts like jars zips and aars)

Reply:
@mikehardy Got it. So why are making it necessary to configure `jetifier`. Is it necessary ?
![image](https://user-images.githubusercontent.com/8683960/65900383-54aefb80-e3cf-11e9-9fb4-de4cd2f48e1f.png)

because, it's creating problem in making build, & that's the reason we need to set these config values... Can we add these as an optional configuration for only case, if user get the exact error ?

Reply:
It depends? If you are using RN59 *and* (a critical *and*) you want to go to AndroidX then you need those in gradle config, and you need a call to `npx jetify`. So that sounds like your case. But maybe you are using RN59 and you want to stay on support libraries. Then you *must not* have those, and you need a call to `npx jetify -r`. And that is the use case for other people. It depends.

Reply:
@mikehardy Alright !